doc/source/myrocks/variables.rst

```diff
@@ -2164,6 +2188,59 @@ Specifies the number if table caches.
 Default value is ``6``.
 Allowed range is from ``0`` to ``19``.
 
+.. variable:: rocksdb_table_stats_background_thread_nice_value
+
+   :version 5.7.30-33: Implemented
+   :cli: ``--rocksdb-table-stats-background-thread-nice-value``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Numeric
+   :default: ``0``
+   
+The nice value for index stats.
+The minimum = -20 (THREAD_PRIO_MIN)
+The maximum = 19 (THREAD_PRIO_MAX)
+
+.. variable:: rocksdb_table_stats_max_num_rows_scanned
+
+   :version 5.7.30-33: Implemented
+   :cli: ``--rocksdb-table-stats-max-num-rows-scanned``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Numeric
+   :default: ``O``
```

Comment:
Should be 0 (zero) instead of O (a letter)

Reply:
fixed table_stats_max_num_rows_scanned